vuepress/docs/next/tutorials/ecr/ecr-bundle-filters.md

```diff
@@ -34,10 +34,6 @@ To enable filtering of a bundle, the custom-resource representing the bundle on
 <td align="left"><p><code>fragment: "true"</code></p></td>
 <td align="left"><p>The bundle contains one or more fragments</p></td>
 </tr>
-<tr class="even">
```

Comment:
@firegloves did we take filters out of the bundle? Or are they just not used in the filter?

Reply:
Pages in the bundle are not processed right now, we have removed that feature. That's the rationale behind this removal

Reply:
No we simply removed the "Page" checkbox. We use CRD file labels, so nothing more has to be removed in order to prevent this functionality.